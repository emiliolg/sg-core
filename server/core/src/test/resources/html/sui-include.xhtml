<html sui-view="tekgenesis.service.html.Views" sui-params="title, content : Html" prefix="og: http://ogp.me/ns#">
    <head>
        <sui-include src="/sui-include-unreferenced-header" pageTitle="@title" pageDescription="This tests multiple params in an include"/>
    </head>

    <body>
        <sui-include src="@content"/>
    </body>
</html>
