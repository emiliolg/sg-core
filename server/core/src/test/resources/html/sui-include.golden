<!DOCTYPE html>
<html prefix="og: http://ogp.me/ns#" sui-scope="/html/sui-include.xhtml/0">
    <head>
        
    <title>Salvador</title>
    <link rel="stylesheet" href="http://cdnd.epi.ninja/sg/concat?/sha/public/css/alpha.css/0125c03ab7&amp;/sha/public/css/beta.css/0a4f910a6e"></link>
    <meta name="description" content="This tests multiple params in an include"></meta>
    <meta charset="utf-8"></meta>

    
<meta property="og:title" content="Salvador betting page"></meta>
<meta property="og:image" content="img/image.jpeg"></meta>
<meta property="og:image:width" content="200"></meta>
<meta property="og:image:height" content="200"></meta>
<meta property="og:image" content="img/image2.jpeg"></meta>
<meta property="og:image:width" content="200"></meta>
<meta property="og:image:height" content="200"></meta></head>

    <body>
        <div sui-scope="/html/sui-include-referenced-content.xhtml/2" class="alpha beta">
    <img src="babies/Salvador"></img>
    <a href="https://www.facebook.com/dialog/oauth?client_id=&amp;redirect_uri=http://tekgenesis.net:8080/sg/fb/login//secure/page&amp;scope=email,user_friends"></a>
</div>
    
<script class="suiScopesCache">
//<![CDATA[
var suiScopesCache = {"/html/sui-include.xhtml/0":{"title":"Salvador"},"/html/sui-include-referenced-content.xhtml/2":{"baby":"Salvador"}};
//]]>
</script>
</body>
</html>